pkg/networking/cilium/cilium_test.go

```diff
@@ -33,34 +32,19 @@ func newCiliumTest(t *testing.T) *ciliumtest {
 		client: *client,
 		h:      h,
 		spec: test.NewClusterSpec(func(s *cluster.Spec) {
-			s.VersionsBundle.Cilium = v1alpha1.CiliumBundle{
-				Cilium: v1alpha1.Image{
-					URI: "public.ecr.aws/isovalent/cilium:v1.9.10-eksa.1",
```

Comment:
Why removing this?

Reply:
because it wasn't getting used, and referring the spec for [upgrader test](https://github.com/aws/eks-anywhere/blob/main/pkg/networking/cilium/upgrader_test.go#L41)

Reply:
I'm ok with that
However, I think the images are being used by the templater when generating the values for helm